### PR TITLE
Setting up project file structure

### PR DESCRIPTION
**Pull request by sayannath**
I have just setup the project file structure. I will be using `ml_collections` to do the config management.

@gustheman @sayakpaul

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Run tests
```
Comment (sayakpaul) [inline .github/workflows/tests.yml]: We can easily discard this test by configuring the local developmental IDE to auto-format code with Black.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+black
+matplotlib
+ml-collections==0.1.0
+pytest==6.2.5
+tensorflow
```
Comment (sayakpaul) [inline requirements.txt]: Specify versions of all the libraries for consistency.

**Review by sayakpaul — CHANGES_REQUESTED** [review]
Structure looks good but really hard to fixate upon without code. 

Additionally, I think it makes sense to move the tests to a separate folder called `tests` for better separation of components.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (sayannath) [reply .github/workflows/tests.yml]: I have configured my VS Code. Will remove this.

**Comment by sayannath** [comment]
Incorporated the changes.

**Review by sayakpaul — APPROVED** [review]
Feel free to merge.

**Pull request merged**